pkg/sql/ir_generator_test.go

```diff
@@ -25,11 +26,21 @@ import (
 	"sqlflow.org/sqlflow/pkg/sql/ir"
 )
 
+func parseOneStatement(statement string) (*parser.SQLFlowSelectStmt, error) {
```

Comment:
This function duplicates with function `ParseOneStatement` in `parser.go`.

Reply:
Thanks for reminding me.